src/index.html

```diff
@@ -20,8 +20,305 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__nav">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="moyo-logo"
+        />
+        <ul class="header__nav-list">
+          <li
+            data-qa="hover"
+            class="header__nav-item"
+          >
+            <a
+              href="#apple"
+              class="header__nav-link header__nav-link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#samsung"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#smartphones"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#lnC"
+              class="header__nav-link header__nav-link--nav-hover"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#gadgets"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#tablets"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#photo"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#video"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </div>
+    </header>
+    <main class="main">
+      <div
+        data-qa="card"
+        class="card-grid"
+      >
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#"
+            class="card__name"
+          >
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </a>
+          <p class="card__code">Product code: 195434</p>
```

Comment:
The product code is repeated across multiple cards. If these are meant to represent different products, ensure each card has a unique product code.